codejailservice/app.py

```diff
@@ -0,0 +1,87 @@
+import os
+import json
+import logging
+
+from codejail import jail_code
+
+from copy import deepcopy
+from flask import Flask, Response, jsonify, request
+
+LOG = logging.getLogger(__name__)
```

Comment:
The LOG is not used, I imagine you imported it just in case

Reply:
If we want to use logging in Flask we need to add the right configuration to get it working right?

Reply:
@jfavellar90  Sure, we need to configure it, do you think we should do that in this MVP ?

Reply:
I think so, at least a raw version that can be improved afterward.